lib/truffle/rbconfig.rb

```diff
@@ -1 +1,205 @@
-# Always loaded
```

Comment:
Can you persuade Git to see this as a move, do you think?

Reply:
In some diff output it showed as a rename, but not on GitHub or in `git log -p`.
Maybe I can try removing `lib/truffle/rbconfig.rb` first.